### DISCUSSION TITLE
plyr compile

### DISCUSSION THREAD

**Discussion by d4lv** [sampotts/plyr#2645] [Q&A]
Hello.. i am trying to do a compile for the plyr but for some reason after the compile the plyr.js (or demo) is not working..

i am using github desktop i open the plyr git bash i install and i make like this the compile https://prnt.sc/pGrHYQ0uQdgP

but after i use the demo.js files the player show this error https://prnt.sc/MCUcksGOo5cr

sorry just i don't know where to ask or to get this type of help :)

**Comment by sampotts** [comment]
I'll close this and track it in the related issue:
https://github.com/sampotts/plyr/issues/2646
